src/bootupd.rs

```diff
@@ -260,7 +260,7 @@ pub(crate) fn update(name: &str, rootcxt: &RootContext) -> Result<ComponentUpdat
 /// daemon implementation of component adoption
 pub(crate) fn adopt_and_update(name: &str, rootcxt: &RootContext) -> Result<ContentMetadata> {
     let sysroot = &rootcxt.sysroot;
-    let mut state = SavedState::load_from_disk(&rootcxt.path)?.unwrap_or_default();
+    let mut state = SavedState::load_from_disk("/")?.unwrap_or_default();
```

Comment:
Why this change?

Reply:
I think we always want to load the saved state from "/boot/bootupd-state.json", right? If running `bootupctl update --from-pending`, my concern is that `RootContext.path` will be `/ostree/deploy/<pending>`, and the saved state will be `/ostree/deploy/<pending>/boot/bootupd-state.json`, so I change it back to "/", but I might be wrong.

Reply:
Yes makes sense!